src/components/GeneralInfo/OperatingSystemCard/OperatingSystemCard.test.js

```diff
@@ -41,30 +49,24 @@ describe('OperatingSystemCard', () => {
 
   it('should render correctly - no data', () => {
     const store = mockStore({ systemProfileStore: {}, entityDetails: {} });
-    const wrapper = mountWithRouter(<OperatingSystemCard store={store} />);
-    expect(toJson(wrapper)).toMatchSnapshot();
+    renderWithRouter(<OperatingSystemCard store={store} />);
+
+    expect(
+      screen
+        .getAllByRole('definition')
+        .reduce((prev, cur) => prev.concat(cur.textContent), '')
+    ).toBe('');
   });
 
   it('should render correctly with data', () => {
     const store = mockStore(initialState);
-    const wrapper = mountWithRouter(<OperatingSystemCard store={store} />);
-    expect(toJson(wrapper)).toMatchSnapshot();
-  });
+    renderWithRouter(<OperatingSystemCard store={store} />);
 
-  it('should render enabled/disabled', () => {
-    const store = mockStore({
-      systemProfileStore: {
-        systemProfile: {
-          loaded: true,
-          ...osTest,
-        },
-      },
-      entityDetails: {
-        entity: {},
-      },
-    });
-    const wrapper = mountWithRouter(<OperatingSystemCard store={store} />);
-    expect(toJson(wrapper)).toMatchSnapshot();
+    expect(
+      screen
+        .getAllByRole('definition')
+        .reduce((prev, cur) => prev.concat(cur.textContent), '')
+    ).toBe('test-releasetest-kerneltest-archNot available0 modules');
```

Comment:
Let's at least separate these with a `,`. :D

Reply:
I will replace these strings with array so that the tests are more readable. I initially departed from the idea to use inline snapshots there, but some issues arose and I ended up with having this weird string 😄 Now after another fresh look this makes more sense to use an array